UnityClient/Assets/Scenes/Map/MapController.cs

```diff
@@ -11,6 +11,7 @@ public class MapController : MonoBehaviour {
     [SerializeField] private Light worldLight;
 
     public MapUiController UIController;
+    public GameEventUI MiniMapController; //Adicionado
```

Comment:
Please use english comments, also this seems irrelevant?

Reply:
Comments removed